api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -124,18 +124,26 @@ type PrecachingStatus struct {
 	Clusters []string          `json:"clusters,omitempty"`
 }
 
+// PolicyContent defines holds the details of an object configured through a Policy
+type PolicyContent struct {
+	Kind      string  `json:"kind,omitempty"`
+	Name      string  `json:"name,omitempty"`
+	Namespace *string `json:"namespace,omitempty"`
+}
+
 // ClusterGroupUpgradeStatus defines the observed state of ClusterGroupUpgrade
 type ClusterGroupUpgradeStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
-	PlacementBindings []string           `json:"placementBindings,omitempty"`
-	PlacementRules    []string           `json:"placementRules,omitempty"`
-	CopiedPolicies    []string           `json:"copiedPolicies,omitempty"`
-	Conditions        []metav1.Condition `json:"conditions,omitempty"`
-	RemediationPlan   [][]string         `json:"remediationPlan,omitempty"`
-	ManagedPoliciesNs map[string]string  `json:"managedPoliciesNs,omitempty"`
-	Status            UpgradeStatus      `json:"status,omitempty"`
-	Precaching        PrecachingStatus   `json:"precaching,omitempty"`
+	PlacementBindings      []string           `json:"placementBindings,omitempty"`
+	PlacementRules         []string           `json:"placementRules,omitempty"`
+	CopiedPolicies         []string           `json:"copiedPolicies,omitempty"`
+	Conditions             []metav1.Condition `json:"conditions,omitempty"`
+	RemediationPlan        [][]string         `json:"remediationPlan,omitempty"`
+	ManagedPoliciesNs      map[string]string  `json:"managedPoliciesNs,omitempty"`
+	ManagedPoliciesContent map[string]string  `json:"managedPoliciesContent,omitempty"`
```

Comment:
If we plan to have one entry for managedpolicies information(name, namespace, complaint status), I think this can be moved to the structure as well. Not ugrent definitely can be a future enhancement :)